addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -60,15 +60,16 @@
                     <PartnerImStatus className="'position-absolute bottom-0 end-0'" partner="notificationRequest.partner"/>
                 </t>
             </NotificationItem>
-            <t t-foreach="displayedPreviews" t-as="preview" t-key="preview.id">
+            <t t-foreach="displayedPreviews" t-as="preview" t-key="preview.thread.id">
```

Comment:
change does not look correct, for channels there could be one item for needaction and one for preview